### PR TITLE
Index show test's

### PR DESCRIPTION
**Pull request by agustinbajo**
https://trello.com/c/YbaCAE34

### CODE REVIEW

File: spec/controllers/api/v1/books_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    context 'When fetching all the books' do
+      let!(:books) { create_list(:book, 3) }
+      before do
+        get :index
+      end
+      it 'responses with the books json' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          books, each_serializer: BookSerializer
+        ).to_json
+        expect(response_body.to_json) =~ JSON.parse(expected)
```
Comment (leoncalermo) [inline spec/controllers/api/v1/books_controller_spec.rb]: here you can use expect().to eq() or something like that  instead of the =~

Comment (leoncalermo) [reply spec/controllers/api/v1/books_controller_spec.rb]: also if you want to tests that the field validations you could use json schemas to validate the results, just as a tip :wink:

File: public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"files":{"application-eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0.js":{"logical_path":"application.js","mtime":"2018-09-27T14:54:48-03:00","size":85961,"digest":"eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0","integrity":"sha256-6q5uk1ojLHEdKTs+Ir0Piphrm+Czn3Cknnspl3rTKfA="},"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css":{"logical_path":"application.css","mtime":"2018-09-27T14:54:48-03:00","size":676,"digest":"f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794","integrity":"sha256-8NcE3uoCnPAAaX4sAYHsFzobR0ZFRm7YQ+te57shV5Q="}},"assets":{"application.js":"application-eaae6e935a232c711d293b3e22bd0f8a986b9be0b39f70a49e7b29977ad329f0.js","application.css":"application-f0d704deea029cf000697e2c0181ec173a1b474645466ed843eb5ee7bb215794.css"}}
```
Comment (leoncalermo) [inline public/assets/.sprockets-manifest-33d6a223bc4d69f1df15bf0a3b3936d9.json:1]: what is this 🤔  ?

**Review by leoncalermo — CHANGES_REQUESTED** [review]

**Comment by agustinbajo** [comment]
### Trello card's of this branch


- https://trello.com/c/YbaCAE34

- https://trello.com/c/OyZkoo3K

**Review by leoncalermo — APPROVED** [review]

**Pull request merged**